prdoc/pr_8585.prdoc

```diff
@@ -0,0 +1,15 @@
+title: fix epmb solution duplicate issue + add remote mining apparatus to epm
+doc:
+- audience: Runtime Dev
+  description: |
+    Prevents the NPoS election process from accepting duplicate voters and targets.
+crates:
+- name: pallet-election-provider-multi-phase
+  bump: minor
+- name: frame-election-provider-solution-type
+  bump: patch
+- name: frame-election-provider-support
+  bump: patch
+- name: sp-npos-elections
+  bump: major
```

Comment:
Can we not re-use another error?  
Either way it should be `minor`, otherwise we will bump crate versions.

Reply:
Nope sorry all other errors are very unrelated. Let's just do minor